tests/core/test_verifying.py

```diff
@@ -189,7 +190,7 @@ def test_ecr(seeder):
         )
         assert result.status == falcon.HTTP_202
         hby.kevers[hab.pre] = hab.kever
-        auth = Authorizer(hby, vdb, eccrdntler.rgy.reger, [LEI1])
+        auth = Authorizer(hby, vdb, eccrdntler.rgy.reger, [LEI1], {"EBA Data Submitter"})
```

Comment:
Would prefer this to use an "EBA Data Submitter" constant here and in the other areas.